src/common/src/session_config/mod.rs

```diff
@@ -34,6 +34,7 @@ const CONFIG_KEYS: [&str; 9] = [
     "RW_BATCH_ENABLE_LOOKUP_JOIN",
     "MAX_SPLIT_RANGE_GAP",
     "SEARCH_PATH",
+    "RW_CHECKPOINT_QUERY",
```

Comment:
I think we can have a better name for this. How about:
- VISIBILITY_MODE == all: all epochs are visible
- VISIBILITY_MODE == checkpoint: only checkpoin epochs are visible